website/docs/addon_usd_resolver_artist.md

```diff
@@ -0,0 +1,19 @@
+---
+id: addon_usd_resolver_artist
+title: AYON USD Resolver
+sidebar_label: USD Resolver
+description: AYON USD Resolver documentations for artists.
+toc_max_heading_level: 5
+---
+
+import ReactMarkdown from "react-markdown";
+import versions from '@site/docs/assets/json/Ayon_addons_version.json'
+
+<ReactMarkdown>
+  {versions.USD_Badge}
+</ReactMarkdown>
+
+
```

Comment:
We will just explain what a resolver is, that these are AYON Entity URIs and that they work using a specialized AYON resolver - that's it.

And explain that there will be generations of pinning files on the farm, so artists are aware of _what that file is_.